src/sonarwhal-theme/layout/partials/rule-category.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="subtitle category-description">
+    <p class="subtitle">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod bibendum laoreet. Proin gravida dolor sit amet lacus accumsan et viverra justo commodo. </p>
```

Comment:
We need a way for the `lorem ipsum` to be pulled from somewhere. Is this for the category's description?

Reply:
I'm going to add the category description to the file `_config.yml`